_maps/shuttles/pirate_default.dmm

```diff
@@ -660,7 +660,6 @@
 	name = "Pirate Ship";
 	port_direction = 8;
 	preferred_direction = 1;
-	timid = 0;
```

Comment:
Have you made sure this still works?

Reply:
I tested a lot of shuttles so I think I did this one but I cant remember atm. I'll check when I get home.